relay-server/src/endpoints/common.rs

```diff
@@ -431,3 +431,24 @@ pub fn create_text_event_id_response(id: Option<EventId>) -> HttpResponse {
         .content_type("text/plain")
         .body(format!("{}", id.0.to_hyphenated()))
 }
+
+/// A helper for creating Actix routes that are resilient against double-slashes
+///
+/// Write `normpath("api/store")` to create a route pattern that matches "/api/store/",
+/// "api//store", "api//store////", etc.
+pub fn normpath(route: &str) -> String {
```

Comment:
I'd prefer if we pass in the leading and trailing slash always to make this more self-documenting on what's the default, and we just make all `+` except for the last one `*`.

Reply:
Also, please add some tests for completeness :)